types/errors/abci_test.go

```diff
@@ -196,21 +189,21 @@ func TestRedact(t *testing.T) {
 
 	for name, tc := range cases {
 		spec := tc
-		t.Run(name, func(t *testing.T) {
-			redacted := Redact(spec.err)
-			if spec.untouched {
-				require.Equal(t, spec.err, redacted)
-			} else {
-				// see if we got the expected redact
-				require.Equal(t, spec.changed, redacted)
-				// make sure the ABCI code did not change
-				require.Equal(t, abciCode(spec.err), abciCode(redacted))
-			}
-		})
+		redacted := Redact(spec.err)
```

Comment:
In general, when should we keep or remove `t.Run()` (or `s.Run()` here)?

Reply:
Suites run within a `testing.T` context. Suites can run in parallel, test cases not just yet